content/shmem_addr_accessible.tex

```diff
@@ -10,23 +10,25 @@
 \end{Csynopsis}
 
 \begin{apiarguments}
-    \apiargument{IN}{addr}{Data object on the local \ac{PE}.}
+    \apiargument{IN}{addr}{Local address of data object to query.}
     \apiargument{IN}{pe}{Integer id of a remote \ac{PE}.}
 \end{apiarguments}
 
 \apidescription{
-    \FUNC{shmem\_addr\_accessible} is a query routine that indicates whether a local
-    address is accessible via \openshmem routines from the specified remote \ac{PE}.
-
+    \FUNC{shmem\_addr\_accessible} is a query routine that indicates whether
+    the address \VAR{addr} can be used to access the given data object on the
+    specified \ac{PE} via \openshmem routines.
+    
     This routine verifies that the data object is symmetric and accessible with
     respect to a remote \ac{PE} via \openshmem data transfer routines.  The
-    specified address \VAR{addr} is a data object on the local \ac{PE}.
+    specified address \VAR{addr} is the local address of the data object on the
+    local \ac{PE}.
 }
 
 \apireturnvalues{
-    The return value is \CONST{1} if \VAR{addr} is a symmetric data object
-    and accessible via \openshmem routines from the specified remote \ac{PE};
-    otherwise, it is \CONST{0}.
+    The return value is \CONST{1} if \VAR{addr} is also a symmetric
```

Comment:
Should we write instead: `if local address \VAR{addr} is also a symmetric address`? In its current form, `also` feels redundant.